benchmarks/sindex.py

```diff
@@ -1,7 +1,18 @@
+import numpy as np
+import random
+
 from geopandas import read_file, datasets
 from geopandas.sindex import VALID_QUERY_PREDICATES
 
 
+# set random seeds for deterministic results
+np.random.seed(0)
```

Comment:
These can be dropped; I don't see that anything below is based on random selections any longer.

Reply:
Good point! Let me test without first.

Reply:
Yep no problems after removing.